packages/react-refresh/src/ReactFreshBabelPlugin.js

```diff
@@ -300,7 +300,20 @@ export default function(babel, opts) {
       }
     });
 
-    const args = [node, t.stringLiteral(key)];
+    let finalKey = key;
+    if (typeof require === 'function' && !opts.emitFullSignatures) {
```

Comment:
(Only because I've been looking for examples for the elvis operator, feel free to ignore)
```suggestion
    if (typeof require === 'function' && !opts?.emitFullSignatures) {
```
Then you wouldn't need the default arguments for `opts`

Reply:
Wanna make sure I won't mess it up in other places where it's ready